servers/rendering/renderer_rd/shaders/scene_forward.glsl

```diff
@@ -2297,6 +2335,14 @@ FRAGMENT_SHADER_CODE
 	}
 #endif
 
+#ifndef LOW_END_MODE
+	if (scene_data.ssao_enabled) {
+		float ssao = texture(sampler2D(ao_buffer, material_samplers[SAMPLER_LINEAR_CLAMP]), screen_uv).r;
+		ao = min(ao, ssao);
```

Comment:
We need to delete SSAO_ao_affect from environment etc.

Reply:
For reference, I opened an issue to track this: https://github.com/godotengine/godot/issues/54611